mypyc/irbuild/ll_builder.py

```diff
@@ -755,6 +756,27 @@ def make_dict(self, key_value_pairs: Sequence[DictEntry], line: int) -> Value:
 
         return result
 
+    def new_list_op(self, values: List[Value], line: int) -> Value:
+        length = self.add(LoadInt(len(values), line, rtype=c_int_rprimitive))
+        empty_list = self.call_c(new_empty_list_op, [length], line)
```

Comment:
Nit: The name of this variable is confusing, since this generally is not an empty (length 0) list.